client/store/users.js

```diff
@@ -35,18 +35,15 @@ export const getAllUsers = () => {
   };
 };
 
-export const getSingleUser = user => {
-  return async dispatch => {
-    try {
-      // console.log('in s u', id);
-
-      const { data } = await axios.get(`/api/users/${user.id}`);
-      dispatch(gotSingleUser(data));
-      console.log('out s u');
-    } catch (error) {
-      console.error(error);
-    }
-  };
+export const getSingleUser = id => async (dispatch, getState) => {
+  try {
+    let userId = id.id;
```

Comment:
after changing the above to this.props.match.params.id, userId should = id from args